src/main/resources/config/application.properties

```diff
@@ -1,4 +1,5 @@
 # Connect Configuration
 connect.issuerUri = https://connect.ch.tudelft.nl/
 connect.clientUri = http://localhost:8080
+spring.template.cache = false
```

Comment:
Ik wel: https://github.com/spring-projects/spring-boot/issues/34
Het doet dus niks in production, daarom had ik het gebruikt. Kan het nog wel in de file erbij commenten, dan gaan mensen er in de toekomst niet aan twijfelen.